### ISSUE TITLE
Player fall timings

### ISSUE THREAD

**Issue by paradox-glitch** [LateClubStudios/NextGenSynopticProject-Outnumbered#35]
- [ ] sometimes the player will start the get up animations before hes hit the ground
- [ ] some times we wait on the ground to long
- [ ] delay after the get up anim is way to long.

**Comment by paradox-glitch** [comment]
**New issue emerged:-**

- [ ] player no longer falls and is already on the floor, this changed after the cut-scene was added.
